.github/workflows/mmv1-rake-tests.yml

```diff
@@ -0,0 +1,39 @@
+name: mmv1-rake-tests
+
+permissions: read-all
+
+on:
+  pull_request:
+    paths:
+      - 'mmv1/**.rb'
```

Comment:
I think this may need to be `mmv1/**/*.rb`

```suggestion
      - 'mmv1/**/*.rb'
```

Reply:
Done